habushu-maven-plugin/src/test/java/org/technologybrewery/habushu/TestSpecifications.java

```diff
@@ -15,6 +13,8 @@
 @Suite
 @IncludeEngines("cucumber")
 @SelectClasspathResource("specifications")
+@ExcludeTags("manual")
+@ConfigurationParameter(key = FILTER_TAGS_PROPERTY_NAME, value = "not manual")
```

Comment:
I: Habushu uses the junit platform suite integrated with cucumber for its testing; I had to include these settings to omit @manual tests.

Reply:
Was able to get rid of the second one